_strategies/modular-procurement.md

```diff
@@ -10,37 +10,37 @@ image-credit: https://www.flickr.com/photos/aaronharmon/
 
-At its core, modular procurement supports iterative development. This means that you are continuously improving, rather than merely hoping for perfection at some point in the future. Building a Minimal Viable Product (MVP) and learning from it ultimately results in a better product than the traditional approach of trying to build a more complex ideal project all at once.
-
-#### There are ways to make sure code is consistent and compatible while using multiple vendors.
-
-There are well-documented, standardized software development practices that can be standardized across vendors, and tools to enforce those. By adopting a "programming style guide" and an automated style checking tool (known as a "linter"), your vendors can ensure the code style is consistent. By adopting other automated quality-checking tools (examples include Code Climate and HoundCI), your vendors can avoid other poor code quality practices. Because the code is open source, vendors can see the code they have to integrate with. By having automated integration and unit tests, your vendors can identify and fix code that breaks or doesn't integrate as soon as it’s written.
+By adopting a "programming style guide" and an automated style checking tool (known as a "linter"), your vendors can ensure the code style is consistent. By adopting other automated quality-checking tools (examples include Code Climate and HoundCI), your vendors can avoid other poor code quality practices. Because the code is open source, vendors can see the code they have to integrate with. By having automated integration and unit tests, your vendors can identify and fix code that breaks or doesn't integrate as soon as it’s written.
   
-
-#### Onboarding vendors is more manageable when projects are modular.
+### Onboarding vendors is more manageable when projects are modular.
 
 One of the major burdens of onboarding is the time for a vendor to understand the entirety of a project. By reducing the size of each project and simplifying the specs, the time required to onboard vendors can be reduced substantially. This makes it manageable to onboard several vendors over the course of a complex project, and reduces the risk.
 
-#### Pre-qualified agile vendor pools can help speed up modular procurement in the long run.
+In modular contracting, however, there will be some amount of system architecture and integration planning by the government to ensure that vendors understand where certain technical and business decisions will impact their portion of the work. Or why previous parts of code that they may interact with are structured that way. 
+
+In particular, with open source software development include clauses in the statement of work that require vendors to make back-ward compatible changes when work developed in service of a particular module should impact the entire project.
+
+### Consider pre-qualified user-centered vendor pools can help speed up modular procurement in the long run.
```

Comment:
suggest: "Consider pre-qualified user-centered vendor pools, which can help speed up modular procurement in the long run."